client/tsconfig.lint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig.json",
+  "exclude": []
+}
```

Comment:
does VSCode pick this up correctly? or do we have to do it the other way round with a tsconfig.prod.json?

Reply:
Well, I don't have any errors in VSCode, but I'm also not very familiar with using it, so I can't say for sure.

Reply:
IDEs will pick up the `tsconfig.json`, so even if it's not relevant now, it can easily lead to errors in the future. Also, I think it's easier to understand if we just have a `tsconfig.prod.json` that excludes the test files.